an2tech-dev/onboarding-platform
# Database seeders with duplicate prevention and data integrity checks

Each seeder now uses firstOrCreate to prevent duplicate entries while maintaining consistent data across multiple seeding runs. 

- Implements firstOrCreate pattern across all seeders
- Adds predefined data for core business entities
- Ensures proper relationships between entities
- Prevents duplicate entries during multiple seeding runs
- Maintains data consistency across companies
- Removes redundant AdminSeeder functionality

## Testing
- Run `php artisan db:seed` multiple times to verify no duplicates
- Verify all relationships are properly established
- Check role assignments for users
- Ensure company-specific data is correctly associated

## Notes
This implementation allows for safe re-running of seeders during development and testing phases without data corruption or duplicate entries.
